source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
                               <specDesc key="handList"/>
                            </specList>
                         </p>
+                        <p>
+                           <specList>
+                              <specDesc key="addDesc"/>
+                           </specList>
+                        </p>
+                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners or librarians: In accordance with standard scholarly practice, these entries are usually indicated in the continuous text with an indication of the location within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```

Comment:
Additions are not only in manuscripts. I have seen manuscript additions to prints (very common), but also print additions to manuscripts (a printed page or two bound in with manuscript leaves). It is probably best to not specify the material.

Reply:
```suggestion
                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners, or librarians. These entries are usually indicated in the continuous text with an indication of the location within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```